pom.xml

```diff
@@ -1583,6 +1587,9 @@
                                     -Xep:StaticOrDefaultInterfaceMethod:OFF
                                     <!-- We generally discourage `var` use. -->
                                     -Xep:Varifier:OFF
+                                    <!-- Yoda conditions are not always more
+                                    readable than the alternative. -->
+                                    -Xep:YodaCondition:OFF
```

Comment:
To be sure: we can discuss enabling this, of course. Maybe I should just get used to it. Even so: the resultant changes also require careful `null` analysis.

Reply:
I'm on the fence about this. Enabling this would result in better/easier readable code IMO. What concerns do you have with enabling this 👀? (Other than getting used to it 😉)

In any case, it would make sense to enable this in a separate PR (especially if we want to make the next internal PSM release).

Reply:
> What concerns do you have with enabling this :eyes: (Other than getting used to it :wink: )

Well the question is whether it's just "getting used to", or that in some cases readability actually suffers. Separately from that it might force us to introduce additional explicit `null` checks, which would be plain awkward.

Reply:
We could open a PR where we enable the check and go over the results and make a decision for EPS there? It is hard to reason about it now if we don't know how "bad" it'll be.

Reply:
Draft PRs are free ;)

That said, I'd also like to see the arguments brought up during the internal discussion on this topic. :student: